internal/doc/butane.yaml

```diff
@@ -344,6 +344,16 @@ root:
               desc: sets the minimum number of pieces required to decrypt the device. Default is 1.
             - name: discard
               desc: whether to issue discard commands to the underlying block device when blocks are freed. Enabling this improves performance and device longevity on SSDs and space utilization on thinly provisioned SAN devices, but leaks information about which disk blocks contain data. If omitted, it defaults to false.
+            - name: enabled 
+              desc: whether or not to use a cex device.
+              transforms:
+                - regex: ".*"
+                  replacement: "UnSupported"
+                  if:
+                    - variant: fcos
+                      min: 1.6.0-experimental
+                    - variant: openshift
+                      min: 4.16.0-experimental
```

Comment:
This was a hack for the GRUB support. Ideally we don't need this.

Reply:
Will remove the `transforms`